book/python/api.md

```diff
@@ -254,8 +254,8 @@ Return a python a python object of the type the next step in the pipeline expect
 A complete TCPSourceDecoder example that decodes messages with a 32-bit unsigned integer _payload\_length_ and a character followed by a 32-bit unsigned int in its _payload_:
 
 ```python
-@decoder(header_length=4, length_fmt=">L")
-def decode(self, bs):
+@wallaroo.decoder(header_length=4, length_fmt=">L")
```

Comment:
We've been using `">I"` in our examples, so this should probably match (they're functionally identical)